wai-aria/role/role_none_conflict_resolution.tentative.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Role None Conflict Resolution Verification Tests</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+    <script src="/wai-aria/scripts/aria-utils.js"></script>
+  </head>
+  <body>
+    
+    <!-- ARIA specs: https://www.w3.org/TR/wai-aria-1.3/#tree_exclusion -->
+    <!-- no conflict, since not focusable; inner button should be exposed as generic - children presentational - //https://github.com/web-platform-tests/interop-accessibility/issues/161 -->
+    <h2>ARIA role="button" nesting ARIA role="button" (no presentational roles conflict resolution, since not focusable)</h2>
+    <div role="button">
+      <span>outer button</span>
+      <span role="button" class="ex-generic" data-testname="non-focusable ARIA[role button] nesting non-focusable ARIA[role button]">Inner button</span>
+    </div>
+
+    <!-- conflict arise, since inner button is focusable - //https://github.com/web-platform-tests/interop-accessibility/issues/161-->
+    <h2>HTML button nesting HTML button</h2>
+    <div id="button-container">
+      <button id="outer-button">
+        <span>outer button</span>
+        <span id="inner-button-container"></span>
+      </button>
+    </div>
+
+    <script>
+      //https://github.com/web-platform-tests/interop-accessibility/issues/161 starts
+      const innerButton = document.createElement("button");
+      innerButton.textContent = " (inner button)";
+      innerButton.id = "inner-button";
+      innerButton.classList.add("ex");
+      innerButton.setAttribute('data-expectedrole','button');
+      innerButton.setAttribute('data-testname','JS injected HTML button within another HTML button');
+      document.getElementById("inner-button-container").appendChild(innerButton);
```

Comment:
Unfortunately, this test may not return anything useful, since `button > * > button` is invalid and not guaranteed to render anything. If it doesn't get a RenderObject, there won't be a reliable way to check the role.

Reply:
I was curious to see the result because button > span > button isn’t allowed in HTML; in fact, as expected, browsers automatically move the nested button outside the parent button. 
However, if you wrap the button using JavaScript, the nested buttons remain and "work as intended".

Reply:
Okay then... Since this is undefined behavior, the only remaining change needed is to remove the expectation that this should be a button... After further WG discussion, we may decide that's right, or it may end up being something else, so I don't want a tentative test (with a potentially invalid assumption) to be the cause of an rendering engine change that needs to be rolled back later.

```js
innerButton.setAttribute('data-expectedrole','SPEC_AMBIGUOUS_LOG_VALUE');
```

I'm marking the PR as approved, pending this last change. Thanks for your diligence and patience!

Reply:
Done!